app/controllers/album.js

```diff
@@ -0,0 +1,32 @@
+const albumService = require('../services/albumService');
+const log = require('../logger');
+
+module.exports = {
```

Comment:
For best practices we can export the functions of controllers directly.
For example, you can do
```
exports.getAlbums = (req,res,next) => {
.
.
.
};

exports.getPhotoOfAlbum = (req,res,next) => {
.
.
.
}
```